configuration/rules-dsl.md

```diff
@@ -178,6 +179,27 @@ then
 end
 ```
 
+### Thing Triggers
+
+You can listen on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific status or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
+```java
+Thing <ThingUid> received update [<status>]
+Thing <ThingUid> changed [from <status>] [to <status>]
+```
+
+The ThingUid is the identifier assigned by OpenHAB to the thing you add. You can find it from
+PaperUI. For example, one of my z-wave device is assigned "zwave:device:c5155aa4:node14".
+
+> Note: You need to use quote around ThingUid. The parser will not recognize the whole text
+> correctly without quotes.
+
+The status used in the trigger and the script is a string (no quotes). You can find all the
+possible values from [Thing Status]({{base}}/concepts/things.html).
```

Comment:
This reference should additionally be found in the first sentence of this section.